lib/src/upgrader.dart

```diff
@@ -358,12 +361,14 @@ class Upgrader {
       return true;
     }
     if (!isUpdateAvailable()) {
+      onNoUpdate?.call();
       return false;
     }
     if (isBlocked) {
       return true;
     }
     if (isTooSoon() || alreadyIgnoredThisVersion()) {
+      onNoUpdate?.call();
```

Comment:
This would be called when an update was available, which makes the meaning of `onNoUpdate` unclear. What does no update actually mean?